magento/pwa-studio
# [PWA-1009] Localize Form Validators

## Description

Localize all strings within form validation of the application

## Related Issue





https://jira.corp.magento.com/browse/PWA-1009

## Acceptance 



### Verification Stakeholders

@dpatil-magento 
@sirugh 
### Specification



### Verification Steps


1. Go to the forms in the app such as Create Account form.
2. Enter all supported invalid scenarios for the form fields.
3. Make sure form error validation messages are translated.

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
